docs/src/getting-started/cash-app-pay.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+title: Cash App Pay
+parent: Getting Started
+nav_order: 4
+---
+
+# Cash App Pay
+{: .d-inline-block .no_toc }
+NEW (v5.3.0)
+{: .label .label-green }
+
+
+<details markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+{: .alert }
+> Cash App Pay is currently available in the following region(s): US
+
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK.
+
+{: .info }
+> When creating a checkout token, both `redirectConfirmUrl` and `redirectCancelUrl` must be set. Failing to do so will cause undefined behavior. The SDK’s example merchant server sets the parameters [here][example-server-props]{:target='_blank'}. See more by checking the [api reference][api-reference-props]{:target='_blank'}.
```

Comment:
![Screenshot 2023-03-03 at 9 29 11 AM](https://user-images.githubusercontent.com/37309039/222787877-ec144963-599f-4ac3-a38d-870f6130d016.png)
The MD did not render correctly

Reply:
Addressed in [1dfda4b](https://github.com/afterpay/sdk-ios/pull/244/commits/1dfda4ba4f5b772d029dc08ef1086237a78cccfb)